### PR TITLE
interface updates for iPhone

### PR DESCRIPTION
**Pull request by Billkaboo**
add, cancel, and done buttons linked
basics of alarm.swift created
nav controllers created and linked

### CODE REVIEW

File: AppleAlarm/AddViewController.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//  Copyright © 2016 Gabriel Isserlis. All rights reserved.
+//
+
+import UIKit
+
+class AddViewController: UIViewController {
+
+    //hello
+    
+    @IBOutlet weak var alarmTitleTF: UITextField!
```
Comment (adm9958) [inline AppleAlarm/AddViewController.swift:15]: Do you really need the TF suffix?

**Comment by antarv** [comment]
lgtm

File: AppleAlarm/AddViewController.swift (hunk 1 above)
Comment (adm9958) [inline AppleAlarm/AddViewController.swift:15]: I'll let it go.  It doesn't really bother me.

**Comment by adm9958** [comment]
LGTM

**Pull request merged**